src/components/dashboard/main-content/course-enrollments/course-cards/InProgressCourseCard.jsx

```diff
@@ -91,6 +86,7 @@ const InProgressCourseCard = ({
     modifyCourseRunStatus({
       status: response.courseRunStatus,
       courseId: response.courseRunId,
+      markedDone: response.markedDone.toLowerCase() === 'true',
```

Comment:
similar comment as above -- should we update API to return a legit boolean so we don't have convert string to boolean?

Reply:
absolutely agreed, shall I do this in two other subtasks, one for api and one for the ui? Perhaps it could also be a great starter ticket for a new team member, just having a thought moment here